packages/herdtools7/herdtools7.7.57/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+synopsis: "The herdtools suite for simulating and studying weak memory models"
+maintainer: "Luc Maranget <Luc.Maranget@inria.fr>"
+authors: [
+  "Luc Maranget <Luc.Maranget@inria.fr>"
+  "Jade Alglave <j.alglave@ucl.ac.uk>"
+]
+homepage: "http://diy.inria.fr/"
+bug-reports: "http://github.com/herd/herdtools7/issues/"
+doc: "http://diy.inria.fr/doc/index.html"
+dev-repo: "git+https://github.com/herd/herdtools7.git"
+license: "CECILL-B"
+build: [make "build" "PREFIX=%{prefix}%"]
+install: [make "install" "PREFIX=%{prefix}%"]
+# @todo Add "build-doc" field
+# @todo Add "build-test" field
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "dune"  {>= "2.7" }
+  "menhir" {>= "20200123"}
+  "zarith"
```

Comment:
```suggestion
  "zarith" {>= "1.10"}
```
This is where Z.divisible was introduced

Reply:
Fixes [Unbound value Z.divisible)](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/d90bb02e7ff9c13d658fbec32511e3b5bc494ca8/variant/compilers,4.09,herdtools7.7.57,lower-bounds)